_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,8 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+* External IPAM support through new Smart Proxy API. Subnets can be now associated with brand new IPAM implementation which offloads IP address management to external entities. An experimental implementaion for phpIPAM is available as Smart Proxy plugin.
```

Comment:
Can you please add a subheading (`####`) for this like we had in previous manuals for headline features? also please add a link(s) for more information - perhaps to the discourse discussion and/or the relevant manual section